Exiled.API/Features/Items/Firearm.cs

```diff
@@ -59,6 +78,203 @@ public Firearm(ItemType type)
         {
         }
 
+        /// <summary>
+        /// Gets all available attachments for all firearms.
+        /// </summary>
+        public static Dictionary<ItemType, AttachmentIdentifier[]> AvailableAttachments => new Dictionary<ItemType, AttachmentIdentifier[]>()
```

Comment:
Since I'm gonna use a `IReadOnlyDictionary`, we don't need to mark as readonly.
`{get;} =` and `=>` is the same thing actually, even if IL compiles different for method bodies.